### PR TITLE
Merge the new printer by abrombo

### LINKED ISSUE #38: Conversion of Printing to Python 3
Thank you for picking up the torch with regard to galgebra.  I just did not have enough time to work on it as required.  I have a question about your conversion to python 3.  I am trying to use your version and it looks like the python 3 print function is not overloaded.  That is if A is a multivector or other object in galgebra the statement "print(A)" does not work.  Is this correct?

### PR DESCRIPTION
**Pull request by utensil**
Fixes #38 .

### CODE REVIEW

**Comment by utensil** [comment]
There's one failed test under Python 3 which is not expected (it works locally), but all failures under Python 2.7 are expected because the original implementation from abrombo didn't consider Python 2 compatibility but I intend to keep it compatible, at least for a while. Will fix later.

**Comment by eric-wieser** [comment]
Notebooks need regenerating

File: examples/LaTeX/curvi_linear_latex.py [hunk 1]
```diff
@@ -92,6 +96,8 @@ def derivatives_in_prolate_spheroidal_coordinates():
     A = ps3d.mv('A','vector',f=True)
     B = ps3d.mv('B','bivector',f=True)
 
+    print('#Derivatives in Prolate Spheroidal Coordinates')
+
```
Comment (eric-wieser) [inline examples/LaTeX/curvi_linear_latex.py:100]: Mind making a new PR with just the changes to this file (and the notebook changes entailed)? I remember being confused by the output without these dividers.

Comment (utensil) [reply examples/LaTeX/curvi_linear_latex.py:100]: OK

Comment (utensil) [reply examples/LaTeX/curvi_linear_latex.py:100]: I'll have to do that even later tonight (need a full dev environment back home)

Comment (eric-wieser) [reply examples/LaTeX/curvi_linear_latex.py:100]: No need to do this any time soon, just might be nice to  pick up at some point.

**Comment by utensil** [comment]
This should be superseded by #510 which merged gprinter from upstream. Feel free to reopen if there are remaining issues.

**Pull request closed without merging**